apps/assessment_registry/filters.py

```diff
@@ -74,3 +75,10 @@ def filter_assessment_registry_issues(self, qs, name, value):
         return qs.filter(
             label__icontains=value
         )
+
+    def filter_is_parent(self, qs, name, value):
+        if not value:
+            return qs
+        return qs.filter(
+            parent__isnull=True
+        )
```

Comment:
I think we may need to do something like this instead to avoid confusion.

```suggestion
        if value is None:
            return qs
        if value:
            # For parent
            return qs.filter(
                parent__isnull=False
            )
        # For child
        return qs.filter(
            parent__isnull=True
        )
```